integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/glue/steps/database/device/DlmsDeviceSteps.java

```diff
@@ -108,29 +114,27 @@
   private final List<SecretBuilder> defaultSecretBuilders =
       Arrays.asList(
           new SecretBuilder()
-              .setSecurityKeyType(E_METER_AUTHENTICATION)
-              .setKey(PlatformSmartmeteringDefaults.SECURITY_KEY_A_DB),
+              .withSecurityKeyType(E_METER_AUTHENTICATION)
```

Comment:
Why not use a Map<>? `securityKeyType` is only used to find the correct builder through a stream.

Reply:
SecretBuilder is stateful. If entries from this list are requested, builder methods are called and then requested by other parts of the test, previous builder values remain in de SecretBuilder instances. Is this intentional?